### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by YING1992** [Seanlinx/mtcnn#46]
thanks for share your code.
I have few questions for training the model.
I try to train P/R/O net with 100000 dataset which include pos, part and neg(2:3:6). However, the results seems bad. I just change the learning rate to 0.00001, other params are keeped. Is my dataset is too small for this problem? If it is possible, can you share your dataset?

**Comment by Seanlinx** [comment]
@YING1992 How did you get your data set, did you follow the instructions in README to generate training set for Rnet and Onet?

**Comment by YING1992** [comment]
thanks for your reply.
At first, in my training, i follow the instructions in README. when i generate onet training set, i get nothing with default params. then,  i just use method like gen_pnet_data.py to generate rnet and onet training set.

**Comment by gehaocool** [comment]
Hi, @Seanlinx , I have some questions about preparing the training data from WiderFace.

I followed the data preparation instruction in README
1. generate data for pnet
2. use gen_hard_example.py to get data for rnet and onet

After these 2 steps, the amount of **rnet** training data is:
pos: 93576, neg: 4856413, part: 279664, is the pos data enough? 

and for **onet**, pos: 167997, neg:34483, part: 79545, is the negative data and part data enough?
I read in another [issue](https://github.com/Seanlinx/mtcnn/issues/28) that you said 60w false positive is needed for rnet, and 8w for onet. I don't know if I get it right, or I set the wrong params in gen_hard_example.py. Shoud I use the similar method of gen_pnet_data.py to generate some 'easy' samples for rnet and onet?

please give me some instruction, thanks!
